pkg/policies/engine/rego/rego.go

```diff
@@ -107,22 +107,28 @@ func (r *Rego) Verify(ctx context.Context, policy *engine.Policy, input []byte,
 	}
 
 	// Try the main rule first
-	if err := executeQuery(mainRule, r.OperatingMode == EnvironmentModeRestrictive); err != nil {
+	if err := executeQuery(resultRule, r.OperatingMode == EnvironmentModeRestrictive); err != nil {
 		return nil, err
 	}
 
 	// If res is nil, it means that the rule hasn't been found
 	if res == nil {
 		// Try with the deprecated main rule
-		if err := executeQuery(deprecatedMainRule, r.OperatingMode == EnvironmentModeRestrictive); err != nil {
+		if err := executeQuery(violationsRule, r.OperatingMode == EnvironmentModeRestrictive); err != nil {
 			return nil, err
 		}
 
 		if res == nil {
-			return nil, fmt.Errorf("failed to evaluate policy: no '%s' nor '%s' rule found", mainRule, deprecatedMainRule)
+			return nil, fmt.Errorf("failed to evaluate policy: neither '%s' nor '%s' rule found", resultRule, violationsRule)
```

Comment:
I actuallyb liked the name of the variable before, it was clearly stating the fact that was deprecated. Now it's not clear if violations is smth we should always have or not